mcc/checker/checkers.py

```diff
@@ -51,11 +51,18 @@ def run_vars(self, blueprint, value):
 
 class CheckCommaSeparated(Checker):
     """
-    Determines if a value is comma-separated or not based on a simple majority
-    count of comma characters versus non-{alphanumeric,comma,space} characters.
+    Determines if a value is comma-separated or not based on several
+    heuristic checks. A string value is considered to be comma-separated if it
+    passes either of the following sub-checks:
+
+        - The value matches a regular expression for comma-delimited strings
+        - The value contains a simple majority count of comma characters versus
+          non-{alphanumeric,comma,space} characters.
+        - The value is a single string with no non-{alphanumeric,comma,space} characters
     """
-    # note this character set is not unicode aware
+    # note this character set is not Unicode aware
     NON_ALPHANUMERIC = re.compile(r'[^A-z0-9 ,]')
+    COMMA_SEPARATED = re.compile(r"^(\s*[A-z0-9\-/<>.\s]+\s*)(,\s*[A-z0-9\-/<>.\s]+\s*)+$")
```

Comment:
Not a show stopper, but I do wonder if we're still being too naive in this approach since it doesn't support things like unicode/escaping sequences. If it's not a major issue for projects though then I think we can deal with it down the line